v2/internal/frontend/assetserver/assetserver.go

```diff
@@ -226,3 +246,12 @@ func (d *AssetServer) logError(message string, args ...interface{}) {
 		d.logger.Error("[AssetServer] "+message, args...)
 	}
 }
+
+func (AssetServer) isRuntimeInjectionMatch(path string) bool {
+	if path == "" {
+		path = "/"
+	}
+
+	return strings.HasSuffix(path, "/") ||
```

Comment:
Do we also need to explicitly check for a `.html` extension?

Reply:
That's only a precondition, it really only injects the runtime in the end if the content-type of the result is 'text/html'. See line 122